cmd/utils/config.go

```diff
@@ -341,126 +342,126 @@ func (kCfg *KlayConfig) SetNodeConfig(ctx *cli.Context) {
 	setAPIConfig(ctx)
 	setNodeUserIdent(ctx, cfg)
 
-	if dbtype := database.DBType(ctx.GlobalString(DbTypeFlag.Name)).ToValid(); len(dbtype) != 0 {
+	if dbtype := database.DBType(ctx.String(DbTypeFlag.Name)).ToValid(); len(dbtype) != 0 {
 		cfg.DBType = dbtype
 	} else {
-		logger.Crit("invalid dbtype", "dbtype", ctx.GlobalString(DbTypeFlag.Name))
+		logger.Crit("invalid dbtype", "dbtype", ctx.String(DbTypeFlag.Name))
 	}
-	cfg.DataDir = ctx.GlobalString(DataDirFlag.Name)
-	cfg.ChainDataDir = ctx.GlobalString(ChainDataDirFlag.Name)
+	cfg.DataDir = ctx.String(DataDirFlag.Name)
+	cfg.ChainDataDir = ctx.String(ChainDataDirFlag.Name)
 
-	if ctx.GlobalIsSet(KeyStoreDirFlag.Name) {
-		cfg.KeyStoreDir = ctx.GlobalString(KeyStoreDirFlag.Name)
+	if ctx.IsSet(KeyStoreDirFlag.Name) {
+		cfg.KeyStoreDir = ctx.String(KeyStoreDirFlag.Name)
 	}
-	if ctx.GlobalIsSet(LightKDFFlag.Name) {
-		cfg.UseLightweightKDF = ctx.GlobalBool(LightKDFFlag.Name)
+	if ctx.Bool(LightKDFFlag.Name) {
+		cfg.UseLightweightKDF = ctx.Bool(LightKDFFlag.Name)
 	}
-	if ctx.GlobalIsSet(RPCNonEthCompatibleFlag.Name) {
-		rpc.NonEthCompatible = ctx.GlobalBool(RPCNonEthCompatibleFlag.Name)
+	if ctx.Bool(RPCNonEthCompatibleFlag.Name) {
```

Comment:
```suggestion
	if ctx.IsSet(RPCNonEthCompatibleFlag.Name) {
```

Reply:
This case makes sense.
```go
cfg.LevelDBBufferPool = !ctx.IsSet(LevelDBNoBufferPoolFlag.Name)
```
But it is difference context we intended. I left these things as it is.

Reply:
Understood. `ctx.Bool` is more accurate than `ctx.IsSet` in the case of `--item=false` syntax.